module/Database/src/Mapper/Meeting.php

```diff
@@ -34,6 +34,34 @@ public function isManaged(MeetingModel $meeting): bool
         return $this->em->getUnitOfWork()->isInIdentityMap($meeting);
     }
 
+    /**
+     * Search for a meeting.
+     *
+     * @return MeetingModel[]
+     */
+    public function searchMeeting(
+        string $query,
+    ): array {
+        $qb = $this->em->createQueryBuilder();
+
+        $fields = [];
+        $fields[] = 'LOWER(m.type)';
+        $fields[] = "' '";
+        $fields[] = 'm.number';
+        $fields[] = "' '";
+        $fields = implode(', ', $fields);
+        $fields = 'CONCAT(' . $fields . ')';
+
+        $qb->select('m')
+            ->from(MeetingModel::class, 'm')
+            ->where($fields . ' LIKE :search')
```

Comment:
I would propose cluttering the search results slightly less, so to find board meeting "BV1234", you can type either "BV1...." or "123...", but not "234". Then you can find GMMs with lower numbers more easy because you don't have all board meetings in between.

Reply:
I have fixed this, but am unsure whether it is a clean/good solution. Quite sure it does work as intended though :)